src/RootContainer.js

```diff
@@ -13,9 +13,11 @@ const RootContainer = ({ serviceUrl, entity }) => {
 
 	useEffect(() => {
 		setLoading(true);
+		const { value } = entity;
+		const valueArr = typeof value === 'string' ? value.split() : value;
```

Comment:
Value should be a number, not a string. I recommend checking if it's an array, and wrapping it (`[value]`) if it isn't.